scanner/src/main/kotlin/scanners/fossid/FossIdNamingProvider.kt

```diff
@@ -35,8 +35,10 @@ import org.ossreviewtoolkit.utils.log
  * Available builtin variables:
  * * **projectName**: The name of the project (i.e. the part of the URL before .git).
  * * **currentTimestamp**: The current time.
+ * * **deltaTag** (scan code only): If delta scans is enabled, this qualify the scan as an *origin* scan or a *delta*
+ * scan.
  */
-class FossIdNamingProvider(
+internal class FossIdNamingProvider(
```

Comment:
This lowering of the visibility is also unrelated to this change, strictly spoken, or?

Reply:
No it is required because `createScanCode` takes a `DeltaTag` in parameters which is `internal` and leads to:
`'public' function exposes its 'internal' parameter type DeltaTag`